src/Nancy.ViewEngines.Razor/CodeParserHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Nancy.ViewEngines.Razor
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+
+    using Nancy.Bootstrapper;
+
+    static class CodeParserHelper

```

Comment:
Why is the class declared as `static` when it does not contain extension methods?